py/client/pydeephaven/_arrow.py

```diff
@@ -57,6 +57,10 @@ def map_arrow_type(arrow_type: pa.DataType) -> Dict[str, str]:
         # if this is a case of timestamp with tz specified
         if isinstance(arrow_type, pa.TimestampType):
             dh_type = "java.time.Instant"
+        if isinstance(arrow_type, pa.Decimal128Type):
+            dh_type = "java.math.BigDecimal"
+        if isinstance(arrow_type, pa.Decimal256Type):
+            dh_type = "java.math.BigDecimal"
```

Comment:
I was hoping that we could add the support of container types (`List`, `DenseUnion`, etc)  here when I filed the ticket https://deephaven.atlassian.net/browse/DH-19185. Does DH-20018 supersede that and we decide not to support these types, or DH-19185 remains open?